service/src/main.rs

```diff
@@ -591,15 +591,13 @@ pub fn init_light_client<E: EnclaveBase + Sidechain>(
 
 	let latest_synced_header = sync_parentchain(enclave_api, api, latest);
 
-	info!("Execute trusted operations for the first time and start side chain block production");
-
-	execute_trusted_calls(enclave_api);
+	enclave_api.trigger_parentchain_block_import().map_err(Error::EnclaveApi)?;
 
```

Comment:
We trigger the import of all parentchain blocks up to the latest one, which we will retain.
Removed executing the trusted calls here, for simplification (they will be executed in the loop immediately after this call anyway).

Reply:
Yes, I think this makes sense.